docs/src/references/mqtt-api.md

```diff
@@ -866,4 +891,26 @@ The alarm severity should be set in the payload.
   </td>
 </tr>
 
+<!-- Health status -->
+<tr>
+  <td>Health status</td>
+  <td>
+    <p>Legacy</p>
+
+```sh
+tedge/health/<child_id>/<service_name>
+```
+
+  <p>New</p>
+
+```sh
+te/<service_topic_id>/status/health
```

Comment:
what is `<service_topic_id>` here?, I feel its better to mention the complete topic here.

Reply:
What about using the more generic `<identifier>` as that is used elsewhere. Also the `<identifier>` could be anything (depending on how the service was registered), so it does NOT have to follow the `te/device/<device-id>/service/<service-name>` schema.

Reply:
I used <service_topic_id> because I wanted it to cover topics which don't follow the default schema as well. We've started to use the term _entity topic id_ regularly now, and I think it makes sense to use _device topic id_ and _service topic id_ for entities which are devices or services, respectively. It's supposed to also cover topic ids which don't follow the default schema.

That being said, I feel that it'd be good to have a _Glossary_ documentation page for all the different terms and acronyms we use. It will be of value to users who learn thin-edge, but also to us, since we can have an authoritative source of information for terms like these.

Reply:
> I used `<service_topic_id>` because I wanted it to cover topics which don't follow the default schema as well. We've started to use the term _entity topic id_ regularly now, and I think it makes sense to use _device topic id_ and _service topic id_ for entities which are devices or services, respectively. It's supposed to also cover topic ids which don't follow the default schema.

Fully agree on that.

> That being said, I feel that it'd be good to have a _Glossary_ documentation page for all the different terms and acronyms we use. It will be of value to users who learn thin-edge, but also to us, since we can have an authoritative source of information for terms like these.

https://github.com/thin-edge/thin-edge.io/issues/2387